Packs/GoogleDrive/pack_metadata.json

```diff
@@ -2,7 +2,7 @@
     "name": "Google Drive",
     "description": "Google Drive allows users to store files on their servers, synchronize files across devices, and share files. This integration helps you to create a new drive, query past activity and view change logs performed by the users, as well as list drives and files, and manage their permissions.",
     "support": "xsoar",
-    "currentVersion": "1.3.3",
```

Comment:
its a minor change so it should be 1.3.4, and please fix the rn file name as well